### DISCUSSION TITLE
Enabling Open Telemetry Support?

### DISCUSSION THREAD

**Discussion by raesene** [moby/moby#47277] [Q&A] [answered]
Hi all, 

I noticed in the 25.0 release notes, there's a line about adding OTel support to Moby/Docker

```
Add OpenTelemetry tracing. https://github.com/moby/moby/pull/45652, https://github.com/moby/moby/pull/45579
```

Just wanted to check, are there any docs on how to enable OTel support on a given Moby/Docker instance? I had a read through the PRs, but wasn't able to immediately tell from there :D

**Comment by cpuguy83** [comment, marked as answer]
These get configured by environment variables for the daemon (typically you'd set this in the systemd unit).

To configure tracer options see the otel docs: https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/

For configuring the trace exporter, see https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/

The key thing is to set OTEL_EXPORTER_OTLP_ENDPOINT
